docs/01-usage/installation.md

```diff
@@ -0,0 +1,29 @@
+# Installation
+
+## Local installation
+
+For [Node modules projects](https://docs.npmjs.com/getting-started/creating-node-modules) we recommend installing **semantic-release** locally and running the `semantic-release` command with [npx](https://www.npmjs.com/package/npx):
+
+```bash
+$ npm install --save-dev semantic-release
+```
+
+Then in the CI environment:
+
+```bash
+$ npx semantic-release
+```
+
+> **Note:** `npx` is a tool bundled with `npm@>=5.2.0`. It is used to conveniently find the semantic-release binary and to execute it. See [What is npx](../05-support/FAQ.md#what-is-npx) for more details.
+
+## Global installation
+
+For other type of projects we recommend installing **semantic-release** directly in the CI environment, also with [npx](https://www.npmjs.com/package/npx):
+
+```bash
+$ npx semantic-release
+```
+
+> **Note:** For a global installation, it's recommended to specify the major **semantic-release** version to install (for example with with `npx semantic-release@15`, or `npm install -g semantic-release@15`). This way your build will not automatically use the next major **semantic-release** release that could possibly break your build. You will have to upgrade manually when a new major version is released.
```

Comment:
I think I would no longer recommend to globally install semantic-release, the `npx` command works great and is supported in all actively maintained Node versions.

Reply:
Not sure I understand this part. Do you mean that I should remove the "Global installation" altogether? What is then the recommendation for CI environments?

Reply:
Yes, remove `npm install -g semantic-release@15` altogether.

For CI environments, install semantic-release as a dev dependency and run `npx semantic-release`, which will use the locally installed version

Reply:
OK. Done in the next push.